### DISCUSSION TITLE
restore deleted organization

### DISCUSSION THREAD

**Discussion by mede-erik** [community/community#38487] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, I accidentally deleted my organization on git hub is there a way to recover it or at least the repositories?

**Comment by david-engelmann** [comment]
You can absolutely recover it. You can email support@github.com and they should be able to help you out. It might take them a bit to respond. [You can also go to this url](https://support.github.com/contact) and select restore deleted data.

**Reply by mede-erik** [reply]
thanks, i contact support,i hope to be able to recover everything

**Reply by david-engelmann** [reply]
@mede-erik Make sure to "Mark an Answer" to close out the discussion

**Reply by Sania-Ijaz** [reply]
I am unable to send an email to this email. What should I do?

**Reply by leonasdev** [reply]
To others facing this issue, I am delighted to inform you that our organization has been successfully restored. All you need to do is open a ticket on support.github.com, and the GitHub support team will kindly assist you.

In my case, aside from re-inviting members and a few basic settings (such as the profile avatar), all repositories and related PRs and issues were successfully recovered.

Special thanks to the GitHub support team for their assistance.

**Reply by vasiliyk** [reply]
> I am unable to send an email to this email. What should I do?

It works only for "GitHub Enterprise"

**Comment by EliasAfara** [comment, marked as answer]
Hey @mede-erik could you please check out the doc for [Restoring a deleted organization](https://docs.github.com/en/enterprise-server@3.4/admin/user-management/managing-organizations-in-your-enterprise/restoring-a-deleted-organization).

**Reply by mede-erik** [reply]
hey, the problem is that I don't have an enterprise account

**Reply by EliasAfara** [reply]
Hmm, If you do not have any backups then it might be more difficult that I thought. 

![image](https://user-images.githubusercontent.com/39487200/200349657-71ed3278-b0e4-4a10-94f8-a1630b7293e6.png)

After 90 days the organization name becomes available for use on a new user or organization account.
You can check out the source [here](https://docs.github.com/en/organizations/managing-organization-settings/deleting-an-organization-account).

I also suggest to contact [support@github.com](mailto:support@github.com), hopefully they could help you out.

**Reply by mede-erik** [reply]
thanks, i contact support, I only have a couple of repo backups, I hope to be able to recover everything

**Reply by EliasAfara** [reply]
Yeah, hopefully everything works out well.

**Reply by ghost** [reply]
@mede-erik were you able to store a deleted organization? I am having the same problem and haven't heard back from support.

**Reply by mede-erik** [reply]
yes I managed but did all the support

**Reply by ghost** [reply]
How did you do all of the support? Did you ever hear back from them? I am in a dilemma. Any advice would be very helpful!

**Reply by mede-erik** [reply]
I just wrote an email to the support then they did everything

**Comment by david-engelmann** [comment]
If github support fails you, I'd try searching for the organization on [the wayback machine](https://archive.org/web/)!

**Reply by mede-erik** [reply]
thanks

**Reply by david-engelmann** [reply]
Oh word

**Comment by ashermason00** [comment]
Just face the same with [16th birthday wishes for son](https://funbirthdaywishes.com/16th-birthday-wishes-for-son/) But You can use this method to restore.  In the top right corner of GitHub.com, click your profile photo, then click Your organizations. Next to the organization, click Settings. In the left sidebar, click Deleted repositories. Next to the repository you want to restore, click Restore. I

**Comment by abhi-iac** [comment]
@mede-erik may I know how long does it took to recover?

**Reply by mede-erik** [reply]
1 week

**Comment by AdminKawldron** [comment]
Hey...  I need help.

My current state:  

-	No repo exists on GitHub.
-	I recently upgraded my account on GitHub. It had a functioning repo, which was connected to Loveable. 
-	I created a GitHub account to be an organisation to make full use of the additional features, and added my existing repository to the organisation.
-	This resulted in the connection between Loveable and GitHub failing, and publishing from Loveable stopped. 
-	To remedy this, I tried to leave the organisation; I couldn’t do this as I was the only user. I had to delete the organisation to remove my user, but this has deleted the repo. 
-	GitHub’s deleted repositories page displays nothing.
-	A repo with the same name currently exists and appears empty (Quick setup), which may be blocking restoration of the original repo/history.
-	I cannot find the deleted repositories in GitHub to restore.
-	The application is fully running in Live.
-	The current version of the application is not running on Loveable. A “Sorry, we ran into an issue starting the live preview!” error message is shown.
-	Loveable displays “An internal error occurred” message whenever I try to give it a prompt.
-	I cannot see the code files in Loveable; it shows an “Error loading files” message.
-	I am unable to restore the application to a previous version in Loveable.  A “I am unable to restore the application to a previous version in Loveable.  A "An error occurred while reverting to this message.” error message is shown.
-	I can see the code changes that were made when that version was published.
-	The previous version of the application is fully running locally on Loveable.

I'm worried that I have to rebuilt the entire frontend again...
